src/sentry/api/endpoints/organization_events_histogram.py

```diff
@@ -51,13 +51,21 @@ def get(self, request: Request, organization) -> Response:
         except NoProjects:
             return Response({})
 
```

Comment:
@wmak does visibility own this endpoint? or is it unowned?

Reply:
Visibility owns this endpoint, I can update codeowners here too goodcatch 🙏